test/ItemsRolesRegistryFacet/itemsFacetTest.ts

```diff
@@ -728,6 +736,39 @@ describe("ItemsRolesRegistryFacet", async () => {
           2
         );
     });
+    it('should NOT transfer an aavegotchi with delegated wearable equipped', async () => {
```

Comment:
We should add the following test: Equip Aavegotchi with 2 delegated wearables and one non-delegated. Remove each wearables one by one and confirm that at each step the Aavegotchi can't be transferred. Only in the last step, when you remove the last delegated wearable, the test should confirm that the owner can transfer the aavegotchi